backends/vulkan/runtime/graph/ops/ExecuteNode.h

```diff
@@ -69,10 +69,12 @@ class ExecuteNode {
     (void)graph;
   }
 
-  virtual inline void trigger_resize(ComputeGraph* graph) {
+  virtual inline bool trigger_resize(ComputeGraph* graph) {
     if (resize_fn_ != nullptr) {
```

Comment:
Inconsistent notation: Once bool operator is used for `std::function` like for: [`pick_shader_fn_`, `pick_local_wg_fn_`](https://github.com/pytorch/executorch/pull/13184/files#diff-5dbb08d9bc7ec1ddf915dc372040d8bdaa377384238d1d4b213688eb04fdb154R109) etc. Here, is explicitly compared to `nullptr`.

Reply:
good points! will update